components/bl-dashlet-component/component.json

```diff
@@ -63,6 +63,18 @@
       "dataBinding": true,
       "handlerLabel": "Width Logic"
     },
+    {
+      "type": "checkbox",
+      "name": "localStorage",
+      "label": "local Storage",
```

Comment:
Local Storage(first letter is capital)

Also, it not look like a boolean variable

Should be something like-  localStorageEnabled